### PR TITLE
Map & List: Differentiation by "types" of facilities

### PR DESCRIPTION
**Pull request by vogelino**
---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1205209947864725

### CODE REVIEW

File: src/components/Card.tsx [hunk 1]
```diff
@@ -21,17 +21,22 @@ export const Card: FC<CardPropsType> = ({
...
       )}
     >
-      <h2 className={classNames(`font-bold text-xl`)}>{title}</h2>
-      {headerContent && <header>{headerContent}</header>}
-      {children}
-      {footer && <footer className="mt-6 w-full">{footer}</footer>}
+      <div className="flex flex-col">
+        <header className={classNames(`flex flex-col pb-2`)}>
+          <h2 className={classNames(`font-bold text-xl`)}>{title}</h2>
+          {headerContent && <header>{headerContent}</header>}
```
Comment (raphael-arce) [inline src/components/Card.tsx:35]: Is this `<header>` inside a `<header>` on purpose?

File: src/lib/hooks/useMapHighlightMarker.ts [hunk 2]
```diff
@@ -20,6 +24,7 @@ export function useMapHighlightMarker(
       'w-10 h-10 bg-primary rounded-full ring-2',
       'ring-offset-white ring-offset-2 ring-primary'
     )
+    customMarker.style.backgroundColor = markerViewport.color
```
Comment (raphael-arce) [inline src/lib/hooks/useMapHighlightMarker.ts:27]: the github warning might be right here, should we add `markerViewport.color` to the effect dependencies?

**Review by raphael-arce — COMMENTED** [review]
Just a couple of questions, otherwise lgtm

File: src/components/Card.tsx (hunk 1 above)
Comment (vogelino) [reply src/components/Card.tsx:35]: You're right! I should fix this! Thx

File: src/lib/hooks/useMapHighlightMarker.ts (hunk 2 above)
Comment (vogelino) [reply src/lib/hooks/useMapHighlightMarker.ts:27]: Will do! Thx! 🙏

**Comment by vogelino** [comment]
@raphael-arce I'm not working on fridays but if you do feel free to review this if you have a minute! Thx! 🙏

**Review by raphael-arce — APPROVED** [review]
LGTM!

**Pull request merged**